### PR TITLE
Add assessment progress bar to outcomes for course

### PR DESCRIPTION
**Pull request by alexandra-aas**
screen shot 2015-12-15 at 10 36 00 am

### CODE REVIEW

**Comment by alexandra-aas** [comment]
DO NOT MERGE! Still need one value fixed.

File: app/views/manage_outcomes/outcomes/_outcomes.html.erb [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <% course.outcomes.each do |outcome| %>
       <tr>
         <td><%= outcome %></td>
-        <td><%= outcome.assessments_count %></td>
+        <td><%= render "progress_bar", amount: outcome.active_assessments_count, total: outcome.active_assessments_count %></td>
```
Comment (alexandra-aas) [inline app/views/manage_outcomes/outcomes/_outcomes.html.erb]: @derekprior I saw that you merged some of the changes we needed to get the progress bar to work. I wasn't sure how to get active assessments count with results or if that is still something you need to do.

Comment (derekprior) [inline app/views/manage_outcomes/outcomes/_outcomes.html.erb]: Working on that now. Wrapping it up before lunch hopefully.

**Comment by derekprior** [comment]
Alrighty, this is complete. Merging.

**Pull request merged**